cmd/dep/testdata/harness_tests/ensure/pkg-errors/case1/testcase.json

```diff
@@ -0,0 +1,7 @@
+{
+   "commands": [
+     ["init"],
```

Comment:
We're about to add a new flag to init in #477 that'll allow us to skip writing out all that help text. That should be merged pretty quickly here - once it is, let's please rebase this PR and change this to `init -no-examples`, and we won't have the unnecessarily long Gopkg.toml file.